backEnd/tasks/taskFactory.js

```diff
@@ -0,0 +1,57 @@
+//this class builds and generates the default tasks when creating a user
+let taskDefaults = require("../tasks.json")
+let dateFormat = require("dateformat");
+
+//these are the tasks that require calculated due dates
+let requiresCalculatedDueDate = new Map([
+  ["BIG Chat", 7],
+  ["BIG Supporters - Family/Partner", 30],
+  ["BIG Supporters - Personal", 30],
+  ["BIG Supporters - Employer", 30],
+  ["BIG Fundamentals", 60],
+  ["BIG Extras - Car Insurance", 60],
+  ["BIG Bio", 60]
+])
+
+//date format
+dateFormat.i18n = {
```

Comment:
Why did you choose to go full date and time for the format? Also a lot of this is already built in to the JS Date() library. Look into toLocaleDateString() and toLocaleString(). They have an options param for like every date format. We should probably just use that instead. 
https://medium.com/swlh/use-tolocaledatestring-to-format-javascript-dates-2959108ea020
(Open in incognito tab if this link is locked behind a paywall)